### PR TITLE
Create portfolio page

### LINKED ISSUE #47: Create portfolio page
**Mockups:** [desktop](https://zpl.io/agY3poO), [tablet](https://zpl.io/amk3N0m), [mobile](https://zpl.io/aBxzD8e)
**URL path:** /portfolio

We should use already prepared components and some hardcoded data to create a responsive view of the portfolio page. DetailsCard and NavBar will be created as separate components so for now, we can replace them with the Box component with white background.

ℹ️ This task is restricted only to the main view. Modal with project details will be created as a separate task

### PR DESCRIPTION
**Pull request by mecies**
### Description

Portfolio page with mock data and missing Navbar and FilterTabs components

- [x] Desktop 
- [x] Tablet
- [x] Mobile

 Closes #47 

**Desktop**

### Screenshots
Screenshot 2020-08-01 at 15 56 36

**Tablet**

Screenshot 2020-08-01 at 21 25 04

**Mobile**

Screenshot 2020-08-01 at 21 24 50

### CODE REVIEW

File: src/pages/portfolio-page/index.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import { PortfolioCard } from '../../components/PortfolioCard';
+
+import { projectData, userData } from './data';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    padding: 0,
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      minHeight: '100vh',
```
Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx]: Why you need this `minHeight`?

File: src/pages/portfolio-page/index.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      padding: theme.spacing(8, 2, 0, 2),
+    },
+  },
+  aside: {
+    display: 'none',
+    [theme.breakpoints.up('lg')]: {
+      display: 'block',
```
Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx:24]: Box is a div, so it has `display: block` by default

File: src/pages/portfolio-page/index.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+    [theme.breakpoints.up('sm')]: {
+      backgroundColor: theme.palette.background.paper,
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      flexDirection: 'column',
+      margin: theme.spacing(0, 0, 7, 0),
+      boxShadow: '0 40px 50px 0 rgba(103, 118, 128, 0.1)',
```
Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx]: Put this shadow to some variable, as you're using it multiple times

File: src/pages/portfolio-page/index.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+      marginBottom: theme.spacing(4),
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      marginBottom: theme.spacing(7),
+      height: 56,
+    },
+  },
+  projectsContainer: {
+    backgroundColor: 'transparent',
```
Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx]: Isn't it transparent by default?

File: src/pages/portfolio-page/index.tsx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    gridRowGap: theme.spacing(4),
+
+    [theme.breakpoints.up('sm')]: {
+      gridRowGap: theme.spacing(3),
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      gridTemplateColumns: '400px 400px',
```
Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx:91]: Wouldn't it be better if we make columns size fluid (1fr 1fr)?

File: src/pages/portfolio-page/index.tsx [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+          <DetailsCard type="desktop" {...userData} />
+        </Box>
+      )}
+      <Box className={classes.main}>
+        <Card className={classes.navbar}>navbar</Card>
+        <Box className={classes.mainContent}>
+          <Box className={classes.projectsContainer}>
+            <SectionTitle className={classes.title}>My works</SectionTitle>
+            <Box className={classes.projects}>
+              {[1, 2, 3, 4, 5, 6].map((_) => (
```
Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx]: Maybye `[...Array(6).keys()]`?

Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx]: Underscore is reserved for unused parameters. Maybe name it `key`?

File: src/pages/portfolio-page/index.tsx [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+                <Box key={_} className={classes.project}>
```
Comment (KonradSzwarc) [inline src/pages/portfolio-page/index.tsx]: Do you have to use Box here? Is it possible to pass this className directly to the PortfolioCard component?

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]

File: src/pages/portfolio-page/index.tsx (hunk 3 above)
Comment (mecies) [reply src/pages/portfolio-page/index.tsx]: Will do 👍

File: src/pages/portfolio-page/index.tsx (hunk 2 above)
Comment (mecies) [reply src/pages/portfolio-page/index.tsx:24]: But it is set to `none` so I have to set it back to `block`. Is there a better way to do it?

File: src/pages/portfolio-page/index.tsx (hunk 1 above)
Comment (mecies) [reply src/pages/portfolio-page/index.tsx]: It was for development purposes and I forgot to remove, thanks for catching

File: src/pages/portfolio-page/index.tsx (hunk 4 above)
Comment (mecies) [reply src/pages/portfolio-page/index.tsx]: It is, will remove 😄

File: src/pages/portfolio-page/index.tsx (hunk 6 above)
Comment (mecies) [reply src/pages/portfolio-page/index.tsx]: It can be, but I think that it doesn't matter. It's a temporary solution until we fetch data

Comment (mecies) [reply src/pages/portfolio-page/index.tsx]: Sure 😄

File: src/pages/portfolio-page/index.tsx (hunk 7 above)
Comment (mecies) [reply src/pages/portfolio-page/index.tsx]: Currently, it's not but I can adjust the component

File: src/pages/portfolio-page/index.tsx (hunk 5 above)
Comment (mecies) [reply src/pages/portfolio-page/index.tsx:91]: Maybe, but it won't change anything since the card width is fixed on desktop😄

**Review by KonradSzwarc — APPROVED** [review]

**Pull request merged**